packages/core/src/rum/instrumentation/resourceTracking/implementation/DatadogRumResource/internalDevResourceBlocklist.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+import type { RUMResource } from '../../domain/interfaces/RumResource';
+
+/**
+ * Logs calls are made every 4ms in dev mode with Expo.
+ * An example URL is http://192.168.1.20:8081/logs
+ */
+const EXPO_DEV_LOGS_REGEX = new RegExp(
```

Comment:
Here you're filtering any request made to an ip address with port and `/logs` path. Even though it seems like a very edge case, couldn't this filter a legitimate network request? Shouldn't we restrict this check to private address ranges: 
- 10.0.0.0 - 10.255.255.255 
- 172.16.0.0 - 172.31.255.255 
- 192.168.0.0 - 192.168.255.255

Reply:
I don't know yet if Expo only ever uses private address ranges for this call. In this case I prefer to be more restrictive and potentially filter out legitimate network requests rather than having 250 requests being made per second.
I'll spend a few minutes digging into Expo code to find out if we can indeed restrict the address range.

Reply:
I understand the Expo code a bit better now, and I get why we are making these requests all the time.

This API call is made every time we do a `console.*` to print it in the packager. As we `console.log` every API call made, we end up in an infinite loop as soon as we make one `console.log` or API call.

Also the network request is always going to be to a local server, and it could potentially be `localhost` as well, so I'm changing the function

Reply:
I changed to `10.*.*.*`, `172.*.*.*` and `192.*.*.*`. I think we can limit even further to match exactly private address ranges, but we would then lose in readability because the Regex will become quite big.
If you think that there is still a chance that we would happen to catch a legitimate network call (I'm not filtering https by the way), I'm happy to change this.